bson/bsoncodec/map_codec.go

```diff
@@ -20,8 +22,9 @@ var defaultMapCodec = NewMapCodec()
 
 // MapCodec is the Codec used for map values.
 type MapCodec struct {
-	DecodeZerosMap   bool
-	EncodeNilAsEmpty bool
+	DecodeZerosMap         bool
+	EncodeNilAsEmpty       bool
+	EncodeKeysWithStringer bool
```

Comment:
Also, doesn't this defaulting to false change the default behavior?

Reply:
went away